Python/initconfig.c

```diff
@@ -63,8 +64,21 @@ static const char usage_3[] = "\
 -W arg : warning control; arg is action:message:category:module:lineno\n\
          also PYTHONWARNINGS=arg\n\
 -x     : skip first line of source, allowing use of non-Unix forms of #!cmd\n\
--X opt : set implementation-specific option. The following options are available:\n\
-\n\
+-X opt : set implementation-specific option (see details with -X help)\n\
+--help-env : print help about Python-specific environment variables\n\
+--check-hash-based-pycs always|default|never :\n\
+         control how Python invalidates hash-based .pyc files\n\
+";
+static const char usage_4[] = "\
+Arguments:\n\
```

Comment:
Moved from line 34; I thought these params deserved their group.  Could even move them before options.

Reply:
@serhiy-storchaka what do you think about moving this arguments section before options?

Reply:
It is up to you.

Reply:
Decided against it, to have initconfig.c stay close to the man page, and respect the classic posix rule of options before arguments.